### DISCUSSION TITLE
7.0.1

### DISCUSSION THREAD

**Discussion by awumii** [awumii/EpicGuard#111] [Announcements]
# 📜 Changelog
* The plugin jar is now split for every platform to decrease the size (caused by including the SQL drivers).
* Many fixes for SQL database issues (invalid syntax, driver not found).
* Improved error logging with additional information.
* Fixed ConcurrentModificationException in name similiarity check.
* Config keys for credentials are now enclosed in quotes by default to prevent error when user forgets to quote them.
* API has been published on JitPack (see README).

⚠ *If you are using MySQL: `use-my-s-q-l` option has been renamed to `use-mysql`. You should rename it before updating or enable mysql again after updating. Your settings.conf file will be automatically updated/fixed anyway. If you don't use MySQL you don't need to do anything*

<hr /><em>This discussion was created from the release <a href='https://github.com/xxneox/EpicGuard/releases/tag/7.0.1'>7.0.1</a>.</em>

**Comment by CyberFlameGO** [comment]
> If you are using MySQL: use-my-s-q-l option has been renamed to use-mysql

Wayyy better
